packages/wonder-blocks-core/components/caaf-aggregator.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import * as React from "react";
+
+type CaaFComponent<P> = {|
+    type: React.ComponentType<P>,
+    props?: P,
+|};
```

Comment:
This is worrisome b/c it means that props won't be specific to each component.

Reply:
I don't follow.